addons/overpressure/XEH_postInit.sqf

```diff
@@ -1,9 +1,13 @@
 #include "script_component.hpp"
 
 ["CBA_settingsInitialized", {
-    TRACE_1("settingsInit eh",GVAR(distanceCoefficient));
-    if (GVAR(distanceCoefficient) <= 0) exitWith {};
```

Comment:
For BWC's sake, don't we want to keep `GVAR(distanceCoefficient` as a setting name?

Reply:
Sorry, it actually makes sense.

Reply:
Atm don't do anything related to this, I want other opinions on this before anything is changed.

Reply:
> Can just add "invalidates old setting" on changelog IMO.

I just don't see a good reason why we should deprecate the setting: The setting type doesn't change, it handles a part of the things it did before and the description of it doesn't change. If people have it set already on servers, changing it would break any of those settings - for what gain?